cmd/loom/loom.go

```diff
@@ -837,6 +837,7 @@ func loadApp(
 		CreateRegistry: createRegistry,
 		Migrations: map[int32]tx_handler.MigrationFunc{
 			1: migrations.DPOSv3Migration,
+			2: migrations.GatewayMigration,
```

Comment:
Each migration is only allowed to run once, so if you want to have separate migrations for regular TG & LOOM TG you'll need:
```golang
2: migrations.GatewayMigration,
3: migrations.GatewayMigration,
```

Reply:
We've decided to run migration against eth gateway only cause we didn't redeploy loom token contract (ERC20 contract)